app/views/modal/list/_item.haml

```diff
@@ -10,9 +10,8 @@
     .media-body
       .list-group-item-heading= list.display_name
       .list-group-item-text.text-muted
-        %span{ id: "#{list.name}-members" }= list.members.count
-        = t 'views.modal.list.members'
+        %span{ id: "#{list.name}-members", data: { count: list.members.count, i18n: t(".members") } }= t(".members", count: list.members.count)
```

Comment:
Having this always be plural rather than "1 member" | "2 members" feels bad. This may also cause problems in other languages.